packages/internal/src/priority-senders.js

```diff
@@ -0,0 +1,85 @@
+import { E, Far } from '@endo/far';
+
+const { Fail, quote: q } = assert;
+
+export const PRIORITY_SENDERS_NAMESPACE_RE = /^[a-zA-Z0-9_-]{1,50}$/;
+
+/**
+ *
+ * @param {ERef<import('./lib-chainStorage.js').StorageNode>} sendersNode
+ */
+export const makePrioritySendersManager = sendersNode => {
+  /**
+   * address to tuple with storage node and set of namespaces that requested priority
+   *
+   * @type {Map<string, readonly [node: StorageNode, namespaces: Set<string>]>}
+   */
+  const addressRecords = new Map();
+
+  const refreshVstorage = (node, namespaces) => {
+    E(node).setValue(
```

Comment:
why not `return` the value? Is the intent for failures to end up as unhandled rejections?

Reply:
fixed